public/docs/ts/latest/guide/_data.json

```diff
@@ -17,7 +17,7 @@
 
   "displaying-data": {
     "title": "Displaying Data",
-    "intro": "Interpolation and other forms of property binding help us show app data in the UI.",
+    "intro": "Property binding shows app data in the UI.",
```

Comment:
Hmmm... I wouldn't say the binding shows it, it just enables showing it. Can you reword this?